### PR TITLE
styling page bug fixes 

### PR DESCRIPTION
**Pull request by egouge**
style bug fixes for themed polygon styles (npe); allow for % in opacity definition for single band raster styles

### CODE REVIEW

**Comment by fgdrf** [comment]
Emily, can you please sign your commit using `git commit -s`. You can easily reset your branch and commit again.

Please also revert the changes in Java-doc comments (@see net.refractions.udig....) in IntervalValuesPanel.java

Do we have any issue filed in JIRA for this bugfix?

**Comment by egouge** [comment]
I think I have fixed the signing and javadocs problems.  I don't have a JIRA issue of these fixes.

File: plugins/org.locationtech.udig.style.advanced/src/org/locationtech/udig/style/advanced/points/widgets/PointLabelsParametersComposite.java [hunk 1]
```diff
@@ -131,8 +131,6 @@ public void init( RuleWrapper ruleWrapper ) {
         GridData labelNameTextGD = new GridData(SWT.FILL, SWT.CENTER, true, false);
         labelNameText.setLayoutData(labelNameTextGD);
         labelNameText.addFocusListener(this);
-        labelNameText.setToolTipText(Messages.LabelNameField_TooltipText);
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.style.advanced/src/org/locationtech/udig/style/advanced/points/widgets/PointLabelsParametersComposite.java]: Is there a reason why the TooltipText has been removed for labelNameText?

**Comment by fgdrf** [comment]
Is there a reason why the TooltipText has been removed for labelNameText in both LineLabelsParametersComposite.java and PointLabelsParametersComposite.java?

Maybe in PolygonLabelParameterComposite.java this tooltip should be set either

`labelNameText.setToolTipText(Messages.LabelNameField_TooltipText);`

BTW: Could you review https://github.com/locationtech/udig-platform/pull/190 for me?

**Comment by fgdrf** [comment]
The Pull-request build is fine at LocationTech Hudson instance : https://hudson.locationtech.org/udig/job/UDIG-PR/6/

**Comment by egouge** [comment]
Nope - I put them back.  I don't know why they got removed.
I will have a look at #190.

**Pull request merged**